NDB.Covid19/NDB.Covid19.iOS/Views/ErrorStatus/ErrorPageViewController.cs

```diff
@@ -110,10 +111,7 @@ private void SetupStyling()
 
 			StyleUtil.InitButtonStyling(ContinueWithSelftestButton, ErrorViewModel.REGISTER_SELFTEST_BUTTON);
 
-			if (ErrorTitle != ErrorViewModel.REGISTER_ERROR_NOMATCH_HEADER)
-            {
-				ContinueWithSelftestButton.Hidden = true;
-			}
+			ContinueWithSelftestButton.Hidden = ErrorTitle != ErrorViewModel.REGISTER_ERROR_NOMATCH_HEADER;
```

Comment:
What about the situations when under 16yo logs in with lookup in MSIS? Shouldn't the app then show the `REGISTER_ERROR_NOMATCH_HEADER` error page but without `ContinueWithSelftestButton` button?

Reply:
The `no-msis` claim is not included in the token from either flows, as the underage check is done on the Verification server before any MSIS lookup is done. So in this change the underage error screen will be the same for both scenarios (no continue button).